### PR TITLE
Fix zeros errors in models created from 3HWC catalog

### PR DESCRIPTION
**Pull request by QRemy**
Fix zeros errors in models created from 3HWC catalog

Requires also: 
https://github.com/gammapy/gammapy-data/pull/26

### CODE REVIEW

**Comment by QRemy** [comment]
Why GitHub action modified the codemeta automatically in this PR and not the previous ones ?

**Review by registerrier — DISMISSED** [review]
Thanks @QRemy.

All errors were incorrect with the previous version?
Why is the new ecsv file needed?

File: gammapy/catalog/hawc.py [hunk 1]
```diff
@@ -263,9 +263,9 @@ def spectral_model(self):
 
         errs = {
             "index": 0.5
-            * (self.data["spec0_index_errp"] + self.data["spec0_index_errn"]),
+            * (np.abs(self.data["spec0_index_errp"]) + np.abs(self.data["spec0_index_errn"])),
```
[the remark this answers is not in the document]
Comment (QRemy) [reply gammapy/catalog/hawc.py:266]: yes where errp=errn

**Comment by registerrier** [comment]
Hi @QRemy the new precision in the HAWC ecsv file is causing some tests to fail. Probably the tolerance should be adapted.

**Comment by registerrier** [comment]
Hi @QRemy , any chance to correct the test fails caused by the new 3HWC ecsv file?

File: gammapy/catalog/hawc.py [hunk 2]
```diff
@@ -263,9 +263,9 @@ def spectral_model(self):
...
             "amplitude": 0.5
-            * (self.data["spec0_dnde_errp"] + self.data["spec0_dnde_errp"]),
+            * (np.abs(self.data["spec0_dnde_errp"]) + np.abs(self.data["spec0_dnde_errp"])),
```
Comment (adonath) [inline gammapy/catalog/hawc.py:268]: Why does this use two time the `..._dnde_errp` column?

File: gammapy/catalog/hawc.py (hunk 1 above)
[the remark this answers is not in the document]
Comment (adonath) [reply gammapy/catalog/hawc.py:266]: If the catalog is reasonably formatted one `np.abs()` call should be sufficient, only for the quantity that is actually negative...

**Review by adonath — CHANGES_REQUESTED** [review]
Thanks @QRemy, I have left some inline comments to address...

**Review by registerrier — APPROVED** [review]
Thanks @QRemy! Looks good.

**Review by adonath — APPROVED** [review]
Thanks @QRemy, no further comments from my side.

**Pull request merged**